### PR TITLE
Use Kapitan Helm dependency type to fetch TopoLVM Helm chart

### PR DESCRIPTION
**Pull request by simu**
This ensures the component continues to work with Kapitan 0.30 (https://github.com/projectsyn/commodore/pull/358)

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ parameters:
     namespace: syn-topolvm
 
     charts:
-      topolvm: topolvm-chart-v3.1.1
+      topolvm: v3.1.1
```
Comment (DebakelOrakel) [inline class/defaults.yml:7]: That should not be changed. As far as i see there is no v3.1.1 tag.

Comment (DebakelOrakel) [reply class/defaults.yml:7]: ah wait, i got it the wrong way around. your changing to the helm repo.

**Review by DebakelOrakel — APPROVED** [review]

**Comment by DebakelOrakel** [comment]
Shall i merge and release this PR?
Because i have an additional change too, will also test it on the lab.

**Comment by simu** [comment]
Feel free to go ahead, this should be a no-op in terms of the resulting catalog manifests

**Pull request merged**